modules/cnf-scheduling-numa-aware-workloads-with-manual-performance-setttings.adoc

```diff
@@ -0,0 +1,185 @@
+metadata:
+  name: numa-deployment-1
+  namespace: openshift-numaresources
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test
+  template:
+    metadata:
+      labels:
+        app: test
+    spec:
+      schedulerName: topo-aware-scheduler <1>
+      containers:
+      - name: ctnr
+        image: quay.io/openshifttest/hello-openshift:openshift
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: "10"
+          requests:
+            memory: "100Mi"
+            cpu: "10"
+      - name: ctnr2
+        image: gcr.io/google_containers/pause-amd64:3.0
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/sh", "-c"]
+        args: [ "while true; do sleep 1h; done;" ]
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: "8"
+          requests:
+            memory: "100Mi"
+            cpu: "8"
+----
+<1> `schedulerName` must match the name of the NUMA-aware scheduler that is deployed in your cluster, for example `topo-aware-scheduler`.
+
+.. Create the `Deployment` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nro-deployment.yaml
+----
+
+.Verification
+
+. Verify that the deployment was successful:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-numaresources
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                READY   STATUS    RESTARTS   AGE
+numa-deployment-1-56954b7b46-pfgw8                  2/2     Running   0          129m
+numaresources-controller-manager-7575848485-bns4s   1/1     Running   0          15h
+numaresourcesoperator-worker-dvj4n                  2/2     Running   0          18h
+numaresourcesoperator-worker-lcg4t                  2/2     Running   0          16h
+secondary-scheduler-56994cf6cf-7qf4q                1/1     Running   0          18h
+----
+
+. Verify that the `topo-aware-scheduler` is scheduling the deployed pod by running the following command:
++
+[source,terminal]
+----
+$ oc describe pod numa-deployment-1-56954b7b46-pfgw8 -n openshift-numaresources
+----
++
+.Example output
+[source,terminal]
+----
+Events:
+  Type    Reason          Age   From                  Message
+  ----    ------          ----  ----                  -------
+  Normal  Scheduled       130m  topo-aware-scheduler  Successfully assigned openshift-numaresources/numa-deployment-1-56954b7b46-pfgw8 to compute-0.example.com
+----
++
+[NOTE]
+====
+Deployments that request more resources than is available for scheduling will fail with a `MinimumReplicasUnavailable` error. The deployment succeeds when the required resources become available. Pods remain in the `Pending` state until the required resources are available.
+====
+
+. Verify that the expected allocated resources are listed for the node. Run the following command:
++
+[source,terminal]
+----
+$ oc describe noderesourcetopologies.topology.node.k8s.io
```

Comment:
In deployment manifest you specify replica:1 which means only 1 deployment pod. 
Before running this cmd you need to check on what Node this pod is running:
oc get pods -n debug -o wide
NAME                               READY   STATUS    RESTARTS   AGE     IP            NODE       NOMINATED NODE   READINESS GATES
numa-deployment-1-958ddbfc-grcqc   2/2     Running   0          6m55s   10.128.2.50   **worker-0**   <none>           <none>


 on then run following cmd:

oc describe noderesourcetopologies.topology.node.k8s.io **worker-0**

Reply:
Makes sense, thanks. Updated the PR.